src/components/venn-diagram/venn-canvas.jsx

```diff
@@ -17,25 +24,23 @@ const VennCanvas = React.createClass({
     window.addEventListener('resize', this._scale);
   },
 
-  getInitialState() {
-    return {
-      width: 1000,
-      height: 1000
-    };
+  componentDidUpdate() {
+    this._scale();
   },
 
   /**
    * Recalculate the venn diagram so that it fits into the canvas
    */
   _scale() {
-    this.setState({
-      width: this.getDOMNode().offsetWidth,
-      height: this.getDOMNode().offsetHeight
-    });
+    let {offsetWidth, offsetHeight} = this.getDOMNode();
```

Comment:
`width` and `height` are used to scale the venn solution (see https://github.com/botify-labs/react-components/blob/enhancement/venn-diagram-onclick/%239/src/components/venn-diagram/venn-canvas.jsx#L71). Google Charts scale themselves automatically.